agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaRemoteEventServiceImpl.java

```diff
@@ -23,6 +23,8 @@ public class AgendaRemoteEventServiceImpl implements AgendaRemoteEventService {
 
   private static final Log         LOG = ExoLogger.getLogger(AgendaRemoteEventServiceImpl.class);
 
+  private static final String      EXCHANGE_CONNECTOR_NAME = "agenda.exchangeCalendar";
```

Comment:
Same remark as here https://github.com/exoplatform/agenda/pull/442/files#r892314539

Reply:
@boubaker is that solution answers to your concern ?